packages/matplotlib/meta.yaml

```diff
@@ -20,15 +20,15 @@ source:
     - - src/html5_canvas_backend.py
       - lib/matplotlib/backends/html5_canvas_backend.py
     - - src/setup.cfg
-      - ./setup.cfg
+      - ./mplsetup.cfg
```

Comment:
FYI, this was the bug, I believe. We can actually drop the freetype patch with this and it still builds (not sure if that's better, but it does build without the patch).

Reply:
If it builds without the patch, I think we should drop it.